src/modules/partition/core/PartitionCoreModule.cpp

```diff
@@ -881,12 +882,18 @@ PartitionCoreModule::initLayout( const QVariantList& config )
             maxSizeString = CalamaresUtils::getString( pentry, "maxSize" );
         }
 
+        if ( pentry.contains( "features" ) )
```

Comment:
Since features are only available to KPMCore 4.1.0 and later, this should have an `#ifdef`fed `cWarning()` if the configuration isn't going to have an effect.